index.js

```diff
@@ -103,7 +103,17 @@ module.exports = function (app, options) {
 
     var locale = this.__getLocale();
     var resource = resources[locale] || {};
-    var text = resource[key] || key;
+
+    // Try to match nested keys, example: model.user.fields.title
```

Comment:
Do this on load localization data maybe better.

app.context[functionName] need not to change.